wildlifelicensing/apps/dashboard/views/officer.py

```diff
@@ -899,7 +900,7 @@ def _render_action(instance):
         elif instance.status == 'draft':
             url = reverse('wl_returns:enter_return', args=(instance.pk,))
             return '<a href="{0}">Edit Return</a>'.format(url)
-        elif instance.status == 'submitted':
+        elif instance.status in ['submitted', 'amended', 'amendment_required']:
```

Comment:
Do we want the officer to be able to curate if it's in a 'amendment_required' state?